src/databricks/labs/ucx/assessment/queries/clusters.sql

```diff
@@ -0,0 +1,9 @@
+-- viz type=table, name=Clusters, columns=cluster_id,cluster_name,creator,compatible,failures
+-- widget title=Clusters, col=0, row=15, size_x=3, size_y=4
+SELECT
+  cluster_id,
+  cluster_name,
+  creator,
+  CASE WHEN success=1 THEN "Compatible" ELSE "Incompatible" END AS compatible,
```

Comment:
Can we get another widget with the top incompatibility reasons across all jobs and clusters? You can use json parse functions for that. 


Eventually this query should expand to pickup the results from a notebook profiler.

Reply:
Let's do that in a follow up PR.